klog.go

```diff
@@ -748,6 +748,35 @@ func (l *loggingT) printWithFileLine(s severity, logr logr.InfoLogger, file stri
 	l.output(s, logr, buf, file, line, alsoToStderr)
 }
 
+// printS if loggr is specified, no need to output with logging module. If
+// err arguments is specified, will call logr.Error, or output to errorLog severity
+func (l *loggingT) printS(err error, loggr interface{}, msg string, keysAndValues ...interface{}) {
+	infoLogr, _ := loggr.(logr.InfoLogger)
```

Comment:
Could you try to avoid casting logger?

Reply:
or t least look at the "ok" return value